monkey-runner/sample/buildSrc/build.gradle

```diff
@@ -0,0 +1,19 @@
+apply plugin: 'groovy'
+
+repositories {
+    jcenter()
+}
+
+// these must mirror the ones in the `plugin` module
+// the dependencies are not transitive (we specify only sourceset below)
+dependencies {
+    compile gradleApi()
+    compile 'com.novoda:gradle-android-command-plugin:1.6.2'
+}
```

Comment:
you could try to factor out this common setup in a `shared.gradle` that could be applied both here and in `plugin/build.gradle`